### PR TITLE
enable multi-arch builds for releases

### PR DESCRIPTION
**Pull request by tpetr**
We've had enough multi-arch branch builds where I feel comfortable with this.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -120,6 +120,7 @@ jobs:
       artifact-name: image-release
       file: Dockerfile
       enable-tests: true
+      multi-arch: true
```
Comment (spencerdrak) [inline .github/workflows/release.yml]: Does this not also need the conditional, or should we always be building multi-arch images and only sometimes be pushing them?

File: .github/workflows/start-release.yml [hunk 2]
```diff
@@ -26,6 +26,11 @@ on:
         type: boolean
         description: Check the box for a dry-run - A dry-run will not push any external state (branches, tags, images, or PyPI packages).
         default: false
+      multi-arch:
```
Comment (spencerdrak) [inline .github/workflows/start-release.yml:29]: This is currently unused, but will be used if we start triggering the `release.yml` directly from this job. Do we want to leave that around until that time?

Comment (tpetr) [reply .github/workflows/start-release.yml:29]: Gotcha, I figure it's fine to leave around. Ultimately I'm going to remove the option to not build multi-arch (just want a little more bake time) so all the `multi-arch`es are temporary

File: .github/workflows/release.yml (hunk 1 above)
Comment (tpetr) [reply .github/workflows/release.yml]: Right, I'll switch to conditional. We'll need build and push to match.

**Review by spencerdrak — APPROVED** [review]

**Pull request merged**